clang/test/CodeGenCXX/fixed-point-zero-init.cpp

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang_cc1 -ffixed-point -S -emit-llvm %s -o - | FileCheck %s
+
+// CHECK: @_ZL1a = internal constant [2 x i32] zeroinitializer
+constexpr _Accum a[2] = {};
```

Comment:
Consider a `test/AST/` test like `static_assert(a[0] == 0 && a[0] != 1);`

This is not supported by -fexperimental-new-constant-interpreter, but I think it is ok.